### ISSUE TITLE
Permissions on user injection

### ISSUE THREAD

**Issue by abr4xas** [Laralum/Laralum#13]
Hi, i have a little problem.

I have:

* super admin
* normal user with only one permission (just update site settings)

With the normal user account clic on `http://localhost:8000/administrator/notifications` from menu top bar:

```php
1/1
HttpException in Handler.php line 133:
This action is unauthorized.
``` 

This is all trace:

```php
in Handler.php line 133
at Handler->prepareException(object(AuthorizationException)) in Handler.php line 109
at Handler->render(object(Request), object(AuthorizationException)) in Handler.php line 47
at Handler->render(object(Request), object(AuthorizationException)) in Pipeline.php line 82
at Pipeline->handleException(object(Request), object(AuthorizationException)) in Pipeline.php line 55
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in LaralumAuth.php line 37
at LaralumAuth->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in LaralumBase.php line 32
at LaralumBase->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in SubstituteBindings.php line 41
at SubstituteBindings->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in VerifyCsrfToken.php line 65
at VerifyCsrfToken->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in ShareErrorsFromSession.php line 49
at ShareErrorsFromSession->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in StartSession.php line 64
at StartSession->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in AddQueuedCookiesToResponse.php line 37
at AddQueuedCookiesToResponse->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in EncryptCookies.php line 59
at EncryptCookies->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in Router.php line 561
at Router->runRouteWithinStack(object(Route), object(Request)) in Router.php line 520
at Router->dispatchToRoute(object(Request)) in Router.php line 498
at Router->dispatch(object(Request)) in Kernel.php line 174
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request)) in Pipeline.php line 30
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in TransformsRequest.php line 30
at TransformsRequest->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in TransformsRequest.php line 30
at TransformsRequest->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in ValidatePostSize.php line 27
at ValidatePostSize->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in CheckForMaintenanceMode.php line 46
at CheckForMaintenanceMode->handle(object(Request), object(Closure)) in Pipeline.php line 148
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in Pipeline.php line 53
at Pipeline->Illuminate\Routing\{closure}(object(Request)) in Pipeline.php line 102
at Pipeline->then(object(Closure)) in Kernel.php line 149
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 116
at Kernel->handle(object(Request)) in index.php line 53
at require_once('/home/abr4xas/html/cblog/public/index.php') in server.php line 21
```

Any help with this :)

**Comment by ConsoleTVs** [comment]
Does the user have laralum::access permission? It's required as well to enter the panel

**Comment by 24aitor** [comment]
Moreover the user that is attemting to access to notifications needs the  	```laralum::notifications.access``` permission.

**Comment by abr4xas** [comment]
@24aitor yes, but my user **only** has `laralum::settings.update`

@ConsoleTVs yes

**Comment by 24aitor** [comment]
And what do you wish? The message appears because the user needs more permissions to access the requested url.

**Comment by abr4xas** [comment]
@24aitor yeah, i get that but i was thinkin that maybe i need add some kind of validation.

Look, i want one user have one permission (`laralum::settings.update`) only that permission. 
The point is, that user can access to that url and i dont need that.

**Comment by ConsoleTVs** [comment]
You mean hide the the entry from the user menu?

**Comment by abr4xas** [comment]
@ConsoleTVs yes, in this case my user dont need that permission so, why show him that option ?

**Comment by ConsoleTVs** [comment]
OK, I get the issue now, This currently works in the sidebar, but the user menu does not have this option yet, it's indeed something to be added, will be added as a feature for the release, thanks!

**Comment by abr4xas** [comment]
Thanks to you... And nice work!!

**Comment by ConsoleTVs** [comment]
You're welcome, keep it up!

**Comment by abr4xas** [comment]
Hi, @ConsoleTVs any update about this? 😄

**Comment by ConsoleTVs** [comment]
Maybe @24aitor can fix it... Im very busy atm

**Comment by 24aitor** [comment]
Hi I've just fixed it on [release 1.2.5](https://github.com/Laralum/Laralum/releases/tag/1.2.5)!

Sorry for the delay and thanks for the feedback!
